incubator/mail2most/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: A Helm chart for Mail2Most a tool for parsing emails to be send to mattermost
+name: mail2most
+version: 1.1.0
+appVersion: "1.0.0"
```

Comment:
can remove the ""

Reply:
oh i see the app version is 1.1.0 and the chart version is 1.0.0 :D ill fix this